src/documents/IoCSS.md

```diff
@@ -4,13 +4,18 @@ layout: default
 category: Infusion
 ---
 
-The Infusion Inversion of Control (IoC) system includes a simple method for distributing options from top-level components through the component tree. This method is dubbed "IoCSS" because it uses a CSS-like syntax.
+The Infusion Inversion of Control (IoC) system includes a simple method for distributing options from any components to any others in the component tree. This system uses 
+a special block named `distributeOptions` which is supported in the options of every Infusion component. The key addressing scheme needed to make this system work &#8212; the
+means for determining where the target of the distribution is &#8212; is dubbed "IoCSS" because it uses a syntax and idiom very similar to that used in [CSS selectors](https://en.wikipedia.org/wiki/Cascading_Style_Sheets#Selector).
 
-## distributeOptions and IoCSS: Downward-Matching CSS-Like Context Selectors For Options Forwarding ##
+## `distributeOptions and IoCSS`: Downward-Matching CSS-Like Context Selectors For Options Forwarding ##
```

Comment:
We might want to change the capitalization of this header to "sentence capitalization"